src/ActionModal/components/ActionModal.vue

```diff
@@ -450,7 +450,7 @@ export default {
 
       if (
         this.network.network_type === "polkadot" &&
-        this.step === feeStep &&
+        (this.step === feeStep || this.step === signStep) &&
```

Comment:
this will trigger now twice and is an expensive process. can we do this just once?

Reply:
I checked and the second time it is triggered it doesn't seem to do anything. Must take it from cache or something like that. But will change it anyway